core/org.wso2.carbon.core/src/main/java/org/wso2/carbon/core/util/KeyStoreManager.java

```diff
@@ -513,10 +672,10 @@ private boolean isCachedKeyStoreValid(String keyStoreName) {
     }
 
     private void updateKeyStoreCache(String keyStoreName, KeyStoreBean keyStoreBean) {
```

Comment:
shall we rename this to `updateTenantKeyStoreCache` as this will only update the tenantKeyStores object and not the customKeyStores?

Reply:
And also let's add a new line after the method signature

Reply:
Fixed in https://github.com/wso2/carbon-kernel/pull/4025/commits/6ea03c474d19b5dd7c6138a60ffb49031621d383